src/sources/soundsourceffmpeg.cpp

```diff
@@ -81,20 +99,38 @@ inline int64_t getStreamStartTime(const AVStream& avStream) {
     auto start_time = avStream.start_time;
```

Comment:
```suggestion
    auto startTime = avStream.start_time;
```

Reply:
The C-style naming was intentional to associate those temporary variables with FFmpeg properties. I don't want to repeat the FFmpeg API docs including units here.